freqtrade/data/dataprovider.py

```diff
@@ -121,7 +121,7 @@ def _emit_df(
                     'type': RPCMessageType.ANALYZED_DF,
                     'data': {
                         'key': pair_key,
-                        'value': dataframe
+                        'value': (dataframe, datetime.now(timezone.utc))
```

Comment:
i think adding a 3rd key would be better.
using tuples kindof assumes that the consumer is python - or has a construct for this.

once it's serialized, it'll be a list - which may or may not maintain order in a different language (and after json serialization).

Reply:
The pair_key is also a tuple